Client/mods/deathmatch/logic/CClientDFF.cpp

```diff
@@ -109,6 +109,10 @@ bool CClientDFF::ReplaceModel(unsigned short usModel, bool bAlphaTransparency)
 
     bool bResult = DoReplaceModel(usModel, bAlphaTransparency);
 
+    // 'Restream' 3D markers
+    if (usModel == 1316 || usModel == 1317 || usModel == 1559)
+        g_pClientGame->ReinitMarkers();
+
```

Comment:
Should be in `DoReplaceModel` method with better DI.

Reply:
Done